curvefs/src/client/rpcclient/metaserver_client.cpp

```diff
@@ -53,7 +53,26 @@ MetaStatusCode MetaServerClientImpl::Init(
 #define RPCTask                                                                \
     [&](LogicPoolID poolID, CopysetID copysetID, PartitionID partitionID,      \
         uint64_t txId, uint64_t applyIndex, brpc::Channel * channel,           \
-        brpc::Controller * cntl) -> int
+        brpc::Controller * cntl, TaskExecutorDone *taskExecutorDone) -> int
+
+#define AsyncRPCTask                                                           \
```

Comment:
AsyncRPCTask is defined same as RPCTask

Reply:
not the same, AsyncRPCTask is "capture  by copy"